libmscore/style.cpp

```diff
@@ -396,6 +396,8 @@ void initStyle(MStyle* s)
          AlignmentFlags::HCENTER | AlignmentFlags::BASELINE, QPointF(0.0, 8.0), OffsetType::SPATIUM, true));
       s->addTextStyle(TextStyle(QT_TRANSLATE_NOOP ("TextStyle", "Technique"), ff, 12, false, true, false,
          AlignmentFlags::LEFT | AlignmentFlags::BASELINE, QPointF(0.0, -2.0), OffsetType::SPATIUM, true));
+      s->addTextStyle(TextStyle(QT_TRANSLATE_NOOP ("TextStyle", "Annotation"), ff, 16, false, true, false,
```

Comment:
i guess we can worry later about what the right defaults are here, but I am pretty sure 16 is bigger than would normally be wanted